packages/backend-common/src/util/KubernetesContainerRunner.ts

```diff
@@ -132,12 +132,17 @@ export class KubernetesContainerRunner implements ContainerRunner {
       imageName,
       command,
       args,
-      logStream = new PassThrough(),
+      logStream,
       mountDirs = {},
       workingDir,
       envVars = {},
     } = options;
 
+    const containerLogStream = new PassThrough();
+    if (logStream) {
```

Comment:
I don't think this is a solution as now we'll have a leak as the streams won't be closed anymore

Reply:
Are you talking about the `containerLogStream` or the passed in `logStream`? The `containerLogStream` should still be closed as that's what previously happened with the original `logStream`. The passed in `logStream` should be closed by the whatever created the `logStream` in my opinion. Though perhaps this can be seen as breaking behavior as the original `logStream` does not get implicitly closed anymore.

Reply:
`outerStream.pipe(innerStream, { end: false })` only makes sure that `innerStream` can't be closed anymore. `outerStream` can still be closed.